### PR TITLE
Added feature to change owner of current organization if the logged in user is admin and owner.

### PR DESCRIPTION
**Pull request by miaoshan**
1. As requested by Tom Lord, this is the change that added "organisation owner" change page which will redirect the user back to "organisation admin" page after successful update. 
2. updated README
3. updated version and footer text

Can you please review these thanks.

### CODE REVIEW

**Comment by birdsarah** [comment]
First thing. In general it's really helpful to split different features into different pull requests - it makes it much easier to review and get little pieces moving forward.

**Comment by birdsarah** [comment]
I know sometimes it can be confusing to do that, happy to help with that if you haven't done it before.

File: django/econsensus/publicweb/forms.py [hunk 1]
```diff
@@ -72,3 +73,12 @@ class FilterForm(forms.Form):
                          initial=EXTRA_CHOICE[0],
                          required=False,
                          widget = forms.Select(attrs={'onchange':'this.form.submit()'}))
+
+class ChangeOwnerForm(forms.ModelForm):
+    class Meta:
+        model = OrganizationOwner
+        fields = ['organization_user']
+    def __init__(self, *args, **kwargs):
+        currentOrgPk = kwargs.pop("currentOrgPk")
```
Comment (birdsarah) [inline django/econsensus/publicweb/forms.py]: it's good practice in python to use "snake case" current_org_pk for variable & method names.

File: django/econsensus/publicweb/urls.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
                     FeedbackSnippetCreate, FeedbackUpdate, \
                     EconsensusActionitemCreateView, EconsensusActionitemUpdateView, \
                     EconsensusActionitemListView, OrganizationRedirectView, \
-                    YourDetails, \
+                    YourDetails, ChangeOwnerView, \
```
Comment (birdsarah) [inline django/econsensus/publicweb/urls.py]: You're quite reasonably following bad habits that have gone before, but each of these imports should be on a separate line. Makes future diffing much easier to read.

File: django/econsensus/publicweb/views.py [hunk 3]
```diff
@@ -805,3 +805,15 @@ def make(cls):
...
+    model = OrganizationOwner
+    form_class = ChangeOwnerForm
+    template_name_suffix = '_update_form'
+    def get_success_url(self):
+        return reverse('organization_admin',
+                kwargs={'organization_pk': self.object.organization.pk})
+    def get_form_kwargs(self):
+        kwargs = super(ChangeOwnerView, self).get_form_kwargs()
+        kwargs.update({'currentOrgPk': self.object.organization.pk})
+        return kwargs
```
Comment (birdsarah) [inline django/econsensus/publicweb/views.py]: pep8 isn't the be all and end all, but it is a good start. what editor do you use? we can probably get you set up with an auto checker.

**Comment by birdsarah** [comment]
This PR needs tests. I would suggest splitting it into the seperate branches for the three different things that are here and then focusing on the tests for the changeowner functionality.

Changing the organization owner is a pretty important thing so we should probably have both unit tests and integration or functional tests.

Cheers,

Bird

**Comment by miaoshan** [comment]
Hi @birdsarah 
Thanks for the feedback.
I have splited changes into three branches and will create pull request respectively. 
I will make sure tests are included in the "change owner" pull request.
closing this pull request now.
Cheers,
Miao

**Comment by miaoshan** [comment]
Hi @birdsarah 
Thanks for the feedback.
I have splited changes into three branches and will create pull request respectively. 
I will make sure tests are included in the "change owner" pull request.
closing this pull request now.
Cheers,
Miao

**Comment by birdsarah** [comment]
Thanks @miaoshan

**Pull request closed without merging**